### ISSUE TITLE
Repeated crashes with PG database on Linux Mint (20) Cinnamon

### ISSUE THREAD

**Issue by kematzy** [dbgate/dbgate#105]
Hey,

I'm just found out about DBGate, got very excited and just installed it on my system.

After install and initial start and connection with a local Postgres database, I **keep on getting repeated crashes asking for a reload**.

The crashes happens even if I leave the app alone (ie: without any user interaction).

The error details shown in the developer console are:

Version: 4.1.10
Build date: 2021-04-27

```js
Unhandler error, checking whether crashed 

PromiseRejectionEvent {isTrusted: true, promise: Promise, reason: TypeError: Cannot read property 'removeChild' of null
    at detach (file:///tmp/.mount_dbgatejTfIR…, type: "unhandledrejection", target: global, …}

// ------------------------------

Uncaught (in promise) TypeError: Cannot read property 'removeChild' of null
    at detach (bundle.js:1)
    at Object.d (bundle.js:24)
    at Object.d (bundle.js:24)
    at Object.d (bundle.js:24)
    at destroy_component (bundle.js:1)
    at Object.d (bundle.js:24)
    at Object.d (bundle.js:24)
    at Object.d (bundle.js:24)
    at destroy_component (bundle.js:1)
    at Object.d (bundle.js:26)
bundle.js:26 
````

Screenshot of the error created.

![2021-0427-162334-Screenshot](https://user-images.githubusercontent.com/6571/116209900-f1d14200-a774-11eb-9b47-b3a6deac9431.png)

I hope the above information is enough to resolve the problem, otherwise, please let me know as I can assist further.

**Comment by kematzy** [comment]
Just to clarify.  I installed the .AppImage version as that was the version pre-selected when clicking the download button.

**Comment by janproch** [comment]
Thanks for report. From the screenshot, I assume, that this crash occurs only when some table is opened. It is possible to put here CREATE TABLE script of this table (possibly without real column names and table name, important are data types and primary/foreign keys), so that I could try to replicate it?

(probably related to this Svelte error https://github.com/sveltejs/svelte/issues/2086 )

**Comment by kematzy** [comment]
Hi Jan,

Thanks for your reply.

I have recorded 3 movies with the app and the shell console below for you to get a better contextual idea of what is happening.

Based upon my interpretation the crash problem is fairly random, but mostly related to switching tables or databases, or loading data.

You can find and download three videos and shell logs, together with a SQL file, at this link: [https://u.pcloud.link/publink/show?code=kZ8BY1XZSMiwxLT57WjUeMsx0wQv28Xcse8X](https://u.pcloud.link/publink/show?code=kZ8BY1XZSMiwxLT57WjUeMsx0wQv28Xcse8X). The link will remain active for the next 2 days.


I apologize for running out of time to explain them further, but please feel free to ask for additional assistance if required.

Thanks for your time and attention.

**Comment by janproch** [comment]
Thank you very much for detailed info. Althought I was not able to replicate this problem, I have fixed 2 issues from videos you provided, the crash should be also fixed. It is part of the newest beta version, release will be next monday. Also I have added to settings posibility to disable background model update - disabling of this prevents appearing of bugs, when user doesn't work with app.

**Comment by kematzy** [comment]
Noted.    I will try the next release and let you know the result.

**Comment by janproch** [comment]
fixed in version 4.1.11. In case of problems please let me know. Closing for now
